src/client/html/style.css

```diff
@@ -0,0 +1,777 @@
+  box-shadow: 0px 4px 0px var(--black);
+  border-radius: 16px;
+  width: 300px;
+  height: 350px;
+}
+
+.team .tiles {
+  height: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+  justify-content: space-evenly;
+  overflow-y: auto;
+}
+
+.tiles .tile {
+  justify-content: center;
+  box-sizing: border-box;
+  width: 105px;
+  height: 100px;
+  border: 2px solid #1f1f1f;
+  filter: drop-shadow(0px 4px 0px #1f1f1f);
+  border-radius: 16px;
+  background-image: linear-gradient(45deg, #121a24, #2b3f56);
+}
+
+.name {
+  z-index: 1;
+  color: var(--white);
+  font-family: var(--font-family);
+  font-weight: 400;
+  font-size: 16px;
+  position: absolute;
+  top: calc(100% - 20px);
+  left: 0;
+  right: 0;
+  text-align: center;
+}
+
+.team .tiles .spymaster {
+  border-color: #f79400 !important;
+}
+
+.team h2 {
+  color: #e2fbfd;
+  font-family: var(--font-family-headings);
+}
+
+.team h3 {
+  color: #e2fbfd;
+  font-family: var(--font-family-headings);
+}
+.active {
+  color: green;
+}
+
+#observers .tiles {
+  width: 100%;
+  display: flex;
+}
+
+#observers {
+  display: flex;
+  flex-direction: row;
+  align-self: center;
+  height: 100px;
+  padding-top: 10px;
+}
+
+#top {
+  display: flex;
+  width: 100%;
+  justify-content: space-evenly;
+  margin-top: 24px;
+}
+
+#gameOver {
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  background-color: rgba(255, 255, 255, 0.4);
+  align-content: center;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#gameOver .container {
+  background: var(--gold);
+  background-size: cover;
+  align-items: center;
+  padding: 16px;
+  margin: 3px;
+  border: 2px solid var(--black);
+  box-shadow: 0px 4px 0px var(--black);
+  border-radius: 16px;
+  color: var(--black);
+  font-weight: 600;
+  width: 200px;
+  height: 150px;
+  color: var(--black);
+}
+
+#gameOver button {
+  background: var(--white);
+  background-size: cover;
+  align-items: center;
+  margin: 3px;
+  width: 100%;
+  height: 42px;
+  border: 2px solid var(--black);
+  box-shadow: 0px 4px 0px var(--black);
+  border-radius: 16px;
+  font-weight: 600;
+  font-size: 20px;
+  font-family: var(--font-family);
+}
+
+#gameOver button:hover {
+  background-color: var(--blue-bright);
+  color: var(--white);
+}
+
+#game h2 {
+  color: var(--white);
+  font-family: var(--font-family-headings);
+}
+
+#game h3 {
+  color: var(--white);
+  font-family: var(--font-family-headings);
+}
+
+#gameStatus {
+  width: 100%;
+  height: 100px;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 24px;
+}
+
+#bottom {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+#board {
+  width: 816px;
+  height: fit-content;
+  justify-content: space-evenly;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 4px;
+  margin: -4px 8px;
+}
+
+.invisible {
+  display: none !important;
```

Comment:
If we have a call, I think we can find a way to exclude any !importants. Usually just means you need to make it higher specificity